packages/wells/src/client/model/WellFilter.ts

```diff
@@ -12,7 +12,7 @@ export interface WellFilter {
    * @type {string}
    * @memberof WellFilter
    */
-  description?: string;
+  stringMatching?: string;
```

Comment:
Is it obvious to everyone that `stringMatching` is `name` and `description` fuzzy search?

Reply:
It was suggested by Juliana we'd use `stringMatching` instead of `query` which is in CDF: 
![image](https://user-images.githubusercontent.com/25263842/106471461-ef2abd80-64a1-11eb-98b3-58ee8582282e.png)

We could change it to something like `matchOnNameOrDescription`, but CDF might expand the search into more fields from what I understand in the description of `query` above.